PROBLEM_3.R

```diff
@@ -0,0 +1,20 @@
+library(tidyverse)
+library(dplyr)
+library(ggplot2)
+setwd("/Users/sofialangs/Documents/GitHub/git-SofiaLangs")
+galaxies <- read_csv("galaxies.csv")
+
+volume_plot<-
```

Comment:
Nice plot! However, if you create a scatter plot of distance vs mass, you will see that small galaxies appear more ofter close to us than far away. This may indicate that small galaxies that are far away havnt been detected.